hyperspy/conftest.py

```diff
@@ -1,12 +1,17 @@
 # Configure mpl and traits to work in a headless system
 from traits.etsconfig.api import ETSConfig
 ETSConfig.toolkit = "null"
-import matplotlib
-matplotlib.use("Agg")
+
+# pytest-mpl 0.7 already import pyplot this line is called, so setting the 
+# matplotlib backend to 'agg' as early as we can will be useless.
+# However, resetting the rcParams to matplotlib default does the job.
+import matplotlib.pyplot as plt
+plt.rcParams.clear()
```

Comment:
As documented, this bit resets the matplotlib backend to 'agg'. Do we want the tests to be run locally with 'agg'? Or would it better to test other backends, which will be the default in the system? The latter could be good to test more stuff but CI will not be necessarily consistent with the tests run locally and it could be a bit confusing if not aware of this detail.